src/Symfony/Component/Workflow/MarkingStore/MethodMarkingStore.php

```diff
@@ -54,7 +54,14 @@ public function getMarking($subject): Marking
             throw new LogicException(sprintf('The method "%s::%s()" does not exist.', \get_class($subject), $method));
         }
 
-        $marking = $subject->{$method}();
+        $marking = null;
+        try {
+            $marking = $subject->{$method}();
+        } catch (\Error $e) {
+            if (!preg_match('/^Typed property ([\w\\\\@]+)::\$(\w+) must not be accessed before initialization$/', $e->getMessage())) {
```

Comment:
shouldn't we also check that the class matches `get_debug_type($subject)`?

Reply:
Fixed

Reply:
what if the property is in a parent class ?